main.py

```diff
@@ -0,0 +1,69 @@
+from typing import List
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import database
+import schemas
+
+app = FastAPI()
+
+
+def get_db():
+    db = database.SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
```

Comment:
database schema should be created automatically
HINT: https://fastapi.tiangolo.com/tutorial/sql-databases/#main-fastapi-app
from database import Base, engine
Base.metadata.create_all(bind=engine)